jenkins/jobs/bmo_e2e_tests.pipeline

```diff
@@ -56,9 +58,11 @@ pipeline {
                       currentBuild.result = "FAILURE"
                   }
                 }
-                archiveArtifacts "artifacts*.tar.gz"
-                /* Clean up */
-                sh "make clean-e2e"
+                timestamps {
+                  archiveArtifacts "artifacts*.tar.gz"
```

Comment:
Do we want archiveArtifacts to be in here? Elsewhere its outside timestamps.

Reply:
no not really I think we are fine without it